demo/c-api/c-api-demo.cc

```diff
@@ -151,13 +156,13 @@ int main(int argc, char** argv) {
   }
   
   // save model
-  const char* fname = "/home/xgb/secure-xgboost/demo/c-api/demo_model.model";
-  safe_xgboost(XGBoosterSaveModel(booster, fname));
+	std::string fname(cwd + "/demo_model.model");
```

Comment:
Small nit: this line looks strangely indented in my editor. Is it the same for you too?

Reply:
Yeah, weird. Amended the commit. We should start using the formatter.